ts/src/index.ts

```diff
@@ -21,6 +21,7 @@ import {perftools} from '../../proto/profile';
 import * as heapProfiler from './heap-profiler';
 import * as timeProfiler from './time-profiler';
 
+export {encode} from './profile-encoder';
```

Comment:
Feels like this change could be a separate PR.

Reply:
The new symbol is being used from the tests, so it could belong in this PR.

Reply:
I have https://github.com/google/pprof-nodejs/pull/14 for this now.
I can refactor on top of it.